pom.xml

```diff
@@ -54,6 +53,7 @@
         <version.syslog4j>0.9.30</version.syslog4j>
         <version.json-simple>1.1.1</version.json-simple>
         <version.undertow>1.3.31.Final</version.undertow>
+        <version.io.undertow>1.3.30.Final</version.io.undertow>
```

Comment:
Can't we use 1.3.31 as in other places? It's a minor bug fix release that fixes some recursion loops bugs...